CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# CONTRIBUTING GUIDELINES
+
+NovelRT is a community-driven project. Contribution is always welcome, and much appreciated.
+It is also how it'll be developed.
+
+First, please take a moment to review our [code of conduct](CODE_OF_CONDUCT.md). We expect that all project participants will adhere to it, and any actions taken that go against the Code of Conduct will not be tolerated.
+
+These guidelines are an attempt at better addressing the incoming issues and pull requests. 
+Please read them closely, and feel free to ask any questions if needed.
+
+## Getting started
+### Submitting Pull Requests
+
+You might know the basics of 
+[contributing on GitHub](https://help.github.com/articles/using-pull-requests) and have a fork
+[properly set up](https://github.com/ohmyzsh/ohmyzsh/wiki/Contribution-Technical-Practices).
+
+You can only create PRs on a dedicated branch.
+
+If you create your own PR, please double check that everything is all right. Also please type what you fixed or added in the PR body,
+[for instance](https://help.github.com/articles/closing-issues-via-commit-messages/)
```

Comment:
I think the message would come across clearer like so:

```suggestion
When creating a PR, please provide an outline of what you fixed or added in the PR body with relevant GitHub keywords and the issue number(s) in question. You can find more information on this [here](https://help.github.com/articles/closing-issues- via-commit-messages/).

Please also ensure that if you add new functionality, and it is testable, that you also add the relevant unit tests, if none exist.

While any feedback that is given from the maintainer team or one of our other contributors should be assumed to be in good faith and for the betterment of the NovelRT project and its sister repositories, please forward any concerns to the relevant places outlined at the beginning of this document.
```

It's more wordy, but I feel this covers the message better. :)
cc @capnkenny thoughts here?

Reply:
Its more concise and direct in this version, @RubyNova